# Question No- 3

## Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.



In [ ]:
#import the libraries
import pandas as pd
import scipy 
import numpy as np
from scipy import stats
from scipy.stats import chi2_contingency

In [6]:
df = pd.read_csv("BuyerRatio.csv")

In [7]:
# Hypthesis testing 
# H0 = All proportions are equal (Given)
# H1 = Not all proportions are equal (Given)

# Which test to be used ?
# We will use Chi-Square test, as we need to see whether there exists relation  between 2 independent qualitative variables which are categorical in nature, i.e., male & female

# About Chi-Square test - Chi-square tests of independence test whether two qualitative variables are independent, that is, whether there exists a relationship between two categorical variables


# alpha = 0.05 as it is given to drwa inferences at 5% significance level

In [8]:
stats.chi2_contingency([df["East"], df["West"], df["North"], df["South"]])

Chi2ContingencyResult(statistic=1.5959455386610577, pvalue=0.6603094907091882, dof=3, expected_freq=array([[  42.76531299,  442.23468701],
       [ 146.81287862, 1518.18712138],
       [ 131.11756787, 1355.88243213],
       [  72.30424052,  747.69575948]]))

## The P value is 0.66 which is greater than alpha, hence we fail to reject the Null hypothesis. 
## This suggests that we accept the null hypothesis. That is All proportions are equal or the male and female buyer rations are similar across the region. 